src/cmd/ksh93/tests/time.exp

```diff
@@ -34,7 +34,7 @@ expect_prompt
 # The times command with no args produces reasonable output.
 log_test_entry
 send "times\r"
-expect -re "\r\nuser$zero_time_re\r\nsys$zero_time_re\r\n" {
+expect -re "\r\n$zero_time_re $zero_time_re\r\n$zero_time_re $zero_time_re\r\n" {
```

Comment:
Can we add a test for non-zero times ?

Reply:
I'd love to. The question is how to do so in a portable fashion that doesn't result in flaky test failures. Note that `zero_time_re` actually allows for a small, less than one second, non-zero value precisely because the shell might have managed to accumulate 10 ms or more of user or system mode time.

Reply:
We don't need to check for specific times, just checking for non-zero value should be sufficient.

Reply:
The problem is that for it to be meaningful we need to ensure the CPU time is always in a predictable range; e.g., [0.5, 0.9] or [1.0, 9.0]. Which, it just occurred to me, isn't hard to do. I can create a CPU bound loop that checks the `SECONDS` var to terminate the loop. That allows us to check that the user mode time for the shell and child processes is correctly reported. I can't think of a simple way to deterministically test the system mode time.